lib/ioki/model/base.rb

```diff
@@ -166,6 +167,18 @@ def reset_attributes!
         end
       end
 
+      # Tries to find a constant in the current modules parent module with the specified name
+      #
+      # class X::Y
+      #   constantize_in_module('Z') # => X::Z
```

Comment:
i'd opt to remove this. I think one can read the code straight forward. you can put it into the PR description. i'd rather not have inline doc.